core/InformationModel/Message/requests/DescriptionRequestMessage_POST.md

```diff
@@ -0,0 +1,103 @@
+# DescriptionRequestMessage POST
```

Comment:
not finished (does it belong to the pull-request-topic?)

Reply:
DRAFT-branch. this was a sketch only!